staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go

```diff
@@ -252,7 +252,8 @@ func (list ErrorList) ToAggregate() utilerrors.Aggregate {
 	return utilerrors.NewAggregate(errs)
 }
 
-func fromAggregate(agg utilerrors.Aggregate) ErrorList {
+// FromAggregate extracts ErrorList from an Aggregate
```

Comment:
Revert this.
API reviewer is going to request that you just use Flatten

Reply:
But `Aggregate` can't be a `*field.Error` without a outer wrap

Reply:
That's fine. This is the same as https://github.com/kubernetes/kubernetes/blob/80be1d6c7217b99fb6ff083a190b44af53a628fc/pkg/scheduler/apis/config/validation/validation_pluginargs.go#L279-L299

Reply:
I am confused. You mean change return type of `nodeSelectorRequirementsAsFieldSelector` from `field.ErrorList` to `error`? Or wrap the `Aggregate` to `field.Error` inside `NewRequirement`? 
I made the [first version](https://github.com/kubernetes/kubernetes/pull/97538/commits/2b4ab5726cc99f585a952eb731bfbd2b74f7a809) ptal, is that what you mean?